File: app/src/main/java/in/woloo/www/trendingblog/BlogDetailsActivity.java
package in.woloo.www.trendingblog;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.RelativeLayout;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import in.woloo.www.R;
import in.woloo.www.utils.Logger;

public class BlogDetailsActivity extends AppCompatActivity {
    private static final String TAG = BlogDetailsActivity.class.getSimpleName();


    @BindView(R.id.topImageBlogDtl)
    ImageView blogImageDtlPage;

    @BindView(R.id.textTitleBlogDtl)
    TextView textTitleBlog;

    @BindView(R.id.titleBlogDtlRel)
    RelativeLayout coinPointRel;

    @BindView(R.id.viaBlogText)
    TextView viaBlogText;

    @BindView(R.id.textDtlArea)
    TextView paragraphText;

    int imgDrawableUri;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_blog_details);
        Logger.i(TAG, "onCreate");
        ButterKnife.bind(this);
        Intent getIntentBlogRcyClick=getIntent();
        if (getIntentBlogRcyClick!=null)
        {
         imgDrawableUri=getIntentBlogRcyClick.getIntExtra("Clicked_blogImage_Position",0);
        }
        System.out.println("Clicked ps url"+imgDrawableUri);
        blogImageDtlPage.setImageResource(imgDrawableUri);
    }
}

File: app/src/main/java/in/woloo/www/more/trendingblog/BlogDetailsActivity.kt
package `in`.woloo.www.more.trendingblog

import android.os.Bundle
import android.widget.ImageView
import android.widget.RelativeLayout
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import butterknife.BindView
import butterknife.ButterKnife
import `in`.woloo.www.R
import `in`.woloo.www.utils.Logger

class BlogDetailsActivity : AppCompatActivity() {
    @JvmField
    @BindView(R.id.topImageBlogDtl)
    var blogImageDtlPage: ImageView? = null

    @JvmField
    @BindView(R.id.textTitleBlogDtl)
    var textTitleBlog: TextView? = null

    @JvmField
    @BindView(R.id.titleBlogDtlRel)
    var coinPointRel: RelativeLayout? = null

    @JvmField
    @BindView(R.id.viaBlogText)
    var viaBlogText: TextView? = null

    @JvmField
    @BindView(R.id.textDtlArea)
    var paragraphText: TextView? = null
    var imgDrawableUri = 0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_blog_details)
        Logger.i(TAG, "onCreate")
        ButterKnife.bind(this)
        val getIntentBlogRcyClick = intent
        if (getIntentBlogRcyClick != null) {
            imgDrawableUri = getIntentBlogRcyClick.getIntExtra("Clicked_blogImage_Position", 0)
        }
        println("Clicked ps url$imgDrawableUri")
        blogImageDtlPage!!.setImageResource(imgDrawableUri)
    }

    companion object {
        private val TAG = BlogDetailsActivity::class.java.simpleName
    }
}